src/ProviderPR/SFA.DAS.ProviderPR/Application/Queries/GetEASUserByEmail/GetEasUserByEmailHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using MediatR;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests.EmployerAccounts;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.EmployerAccounts;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.ProviderPR.Application.Queries.GetEasUserByEmail;
+public class GetEasUserByEmailQueryHandler(IAccountsApiClient<AccountsConfiguration> apiClient) : IRequestHandler<GetEasUserByEmailQuery, GetEasUserByEmailQueryResult>
+{
+
+    public async Task<GetEasUserByEmailQueryResult> Handle(GetEasUserByEmailQuery request, CancellationToken cancellationToken)
+    {
+        var result = await apiClient.Get<GetUserByEmailResponse>(new GetUserByEmailRequest(request.Email));
+
+        var userRef = result?.Ref;
```

Comment:
I would check if the result is null first and exit out if it is rather than checking on userRef.

Reply:
ok